utils/src/server/rpc_module.rs

```diff
@@ -242,8 +246,45 @@ impl SubscriptionSink {
 				params: JsonRpcNotificationParams { subscription: *sub_id, result: &*result },
 			})?;
 
-			// Track broken connections
-			if sender.unbounded_send(msg).is_err() {
+			// Log broken connections
```

Comment:
```suggestion
			// Track broken connections
```

We're not logging anything I think?